src/app/settings/action-triggers/components/action-triggers-list/action-triggers-list.component.ts

```diff
@@ -0,0 +1,241 @@
+import { Component, EventEmitter, Input, OnChanges, Output, SimpleChange, SimpleChanges } from '@angular/core'
+import { IconButton, IconButtonSize } from 'src/app/shared/enums/icon-button'
+import { ActionTrigger, ActionTriggerWithActionInfo } from 'src/app/shared/models/action-trigger'
+import { SofieDroppdownOption } from 'src/app/shared/components/dropdown-button/dropdown-button.component'
+import { DialogService } from 'src/app/shared/services/dialog.service'
+import { ActionTriggerService } from 'src/app/shared/abstractions/action-trigger.service'
+import { HttpFileDownloadService } from 'src/app/core/services/http/http-file-download.service'
+import { ActionTriggerSortKeys, KeyboardTriggerData } from 'src/app/shared/models/keyboard-trigger'
+import { SortOrder } from 'src/app/shared/models/forms'
+
+export enum UserActionsWithSelectedTriggers {
```

Comment:
That was rename to `UserActionsWithSelected`, because in to the comment above on @LindvedKrvang he explain me, that will be used everywhere in setting pages and is good to have general name.  I will rename that to `ActionsWithSelected`